### DISCUSSION TITLE
Make build issue

### DISCUSSION THREAD

**Discussion by Opswatch** [amazeeio/lagoon#1934] [General] [answered]
Hi 
Im trying to run lagoon locally 
Running on a kvm vm/ ubuntu 18.04, using docker with docker compose installed
Git cloned the repo 
Ran make build and got the following error:

docker build --quiet --build-arg LAGOON_VERSION=development --build-arg IMAGE_REPO=lagoon --build-arg ALPINE_VERSION=3.11 -t lagoon/oc -f images/oc/Dockerfile images/oc
Sending build context to Docker daemon   5.12kB
Step 1/21 : ARG ALPINE_VERSION
Step 2/21 : ARG IMAGE_REPO
Step 3/21 : FROM ${IMAGE_REPO:-lagoon}/commons as commons
pull access denied for lagoon/commons, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
Makefile:169: recipe for target 'build/oc' failed
make: *** [build/oc] Error 1

Did I miss a step or variable somewhere?, looks like its trying to pull from a private internet docker registry?

Thanks.

**Comment by tobybellwood** [comment, marked as answer]
In the Make build process, it should have tried to make the lagoon/commons image first (as per https://github.com/amazeeio/lagoon/blob/master/Makefile#L197)

Do you have the lagoon/commons image on your local docker image registry?

If you want to check and retry:
`make clean` - removes the temp files we touch to show an image has been built, and then
`make build` - or if you've got moar power `make -j6 build` to build in parallel.

You can always build the commons image manually `make build/commons` - but I'm not sure why this didn't work in your case?

**Comment by Opswatch** [comment]
Interesting, I'll try and build the image manually, it was a first run,
it's on regular docker network, which is 172. I believe but the
instructions say the registry is at 192. Would that make a difference?,
That would make sense if I was running on virtual box maybe.

Update: make clean got it going currently building the images.

**Comment by Opswatch** [comment]
Containers built properly, they all seem to be running, no error logs from what I can see but when accessing the lagoon ui I get this:
Not Authenticated

Please wait while we log you in...
Any Ideas?, I took a look at the keycloak container logs and it looks good no errors, same with the ui, im able to access the keycloak ui as well with no issues. Did a make clean and rebuilt the containers just in case same issue.
